superset-frontend/src/filters/components/Range/index.ts

```diff
@@ -25,7 +25,7 @@ import thumbnail from './images/thumbnail.png';
 export default class AntdRangeFilterPlugin extends ChartPlugin {
   constructor() {
     const metadata = new ChartMetadata({
-      name: t('Range filter plugin'),
+      name: t('Range filter'),
```

Comment:
Slightly unrelated, but while we're at it I think we should drop the `Antd` prefix from the class names and just go with `RangeFilterPlugin` (same for the others).